### DISCUSSION TITLE
Can a corporation produce the site without commercial use?

### DISCUSSION THREAD

**Discussion by ghost** [mozilla/hubs#5518] [General Questions & Discussion] [answered]
We make a meteverse content and IT promotion in Japan.

We plan Metaverse the Japanese countryside with our clients.
So we are looking for a platform that can meet the following requirements.

Is a corporate contract necessary? If so, how much would a half-year to annual contract cost?
Can a corporation produce the site without commercial use?

**Comment by misslivirose** [comment, marked as answer]
The terms for using Hubs vary depending on what exactly you are using.

The Hubs client code base is released under the [Mozilla Public License 2.0](https://github.com/mozilla/hubs/blob/master/LICENSE). 
Our terms of use for hubs.mozilla.com are here: https://www.mozilla.org/en-US/about/legal/terms/hubs/. 
The terms of use for Hubs Cloud specifically are [here](https://d7umqicpi7263.cloudfront.net/eula/product/94ec4ef1-285d-4e63-b2eb-0b65ea93b942/4fbc708f-d872-4ba0-83b3-8e6385dd9703.pdf).

**Reply by ghost** [reply]
Thank you.
I read the link you sent me.

I found the statement that it can be used for commercial purposes, and I feel the possibility.

https://hubs.mozilla.com/spoke
I'm thinking of creating content using this service.

I was able to find a statement that it was available for commercial use, but I couldn't find a statement that payment was required.
Is it safe to assume that there are no fees?
